src/backend/api/allMeals.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (req, res) => {
+    try {
+        const results = await knex.raw('SELECT id,title FROM meal ORDER BY id;')
+    
+        res.json(results[0]);
```

Comment:
I would suggest checking whether results contains at least one element. That is always a good idea before a statement like `results[0]`